charts/dremio-cluster/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: dremio-module-chart
+name: dremio-chart
 description: A Helm chart for installing a module for Fybrik in Kubernetes
```

Comment:
A Helm chart for installing a **Dremio** module for Fybrik in Kubernetes

Reply:
Fixed